lib/git_reflow/commands/refresh.rb

```diff
@@ -13,14 +13,13 @@
   c.arg_name '[remote_location] - remote repository name to fetch updates from (origin by default),
           [base_branch] - branch that you want to merge with (master by default)'
   c.flag [:r,:remote], default_value: 'origin'
-  c.flag [:b,:branch], default_value: 'master'
+  c.flag [:b,:base], default_value: 'master'
   c.action do |global_options, options, args|
     # usage: git reflow refresh [remote_location - defaults to origin] [base_branch_name - defaults to master]
```

Comment:
I rewrote these comments. They are quite useful for someone just trying to read the code.